src/main/scala/com/microsoft/cdm/write/CDMDataWriterFactory.scala

```diff
@@ -21,7 +21,7 @@ class CDMDataWriterFactory(var adlProvider: ADLGen2Provider,
 
   // TODO: error handling. we're basically assuming successful writes. Need to add logic to remove/rewrite files on failure.
 
-  def createDataWriter(partitionId: Int, attemptNumber: Int): DataWriter[Row] = {
+  def createDataWriter(partitionId: Int, taskId: Long, epochId: Long): DataWriter[Row] = {
```

Comment:
Looks like these arguments are not used inside?

Reply:
Any change to have this new version next week?